src/components/Router/StepComponentMap.ts

```diff
@@ -109,6 +123,10 @@ const buildCaptureStepComponents = (
     welcome: [Welcome],
     userConsent: [UserConsent],
     face: buildFaceComponents(faceStep, deviceHasCameraSupport, mobileFlow),
+    //@ts-ignore
```

Comment:
You don't need `@ts-ignore` here. Your `LazyAuth` component is TypeScript-friendly enough. All the other `@ts-ignore`'s in this file are just a workaround while waiting for the related components to be ported to TS.
```suggestion
```